scripts/bundle-esm.js

```diff
@@ -3,31 +3,50 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-const path = require("path");
-const helpers = require("monaco-plugin-helpers");
+const path = require('path');
+const helpers = require('monaco-plugin-helpers');
 
-const REPO_ROOT = path.join(__dirname, "../");
+const REPO_ROOT = path.join(__dirname, '../');
 
 helpers.packageESM({
   repoRoot: REPO_ROOT,
-  esmSource: "out/esm",
-  esmDestination: "lib/esm",
-  entryPoints: ["monaco.contribution.js", "yamlMode.js", "yaml.worker.js"],
+  esmSource: 'out/esm',
+  esmDestination: 'lib/esm',
```

Comment:
Would it be possible to keep it without esm? Or we are doing the same pattern in other monaco plugins?

Reply:
yes its the same process in other monaco plugins too, except the `lib` folder is called `release`. It is very similar to the [`monaco-json`](https://github.com/microsoft/monaco-json) repo, which this repo is very similar in nature to.